pkg/cloud/gen/main.go

```diff
@@ -985,6 +985,8 @@ func (g *{{.GCEWrapType}}) AggregatedList(ctx context.Context, fl *filter.F) (ma
 		klog.V(4).Infof("{{.GCEWrapType}}.AggregatedList(%v, %v) = %v, %v", ctx, fl, nil, err)
 		return nil, err
 	}
+	g.s.RateLimiter.Observe(ctx, nil, ck)
```

Comment:
Is this path missing callObserverEnd as well?

Reply:
Right, thanks for pointing out, fixed